File: aspnet_server/DurianNet/Models/DataModels/DurianProfile.cs
using DurianNet.Repository;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DurianNet.Models.DataModels
{
    [Table("DurianProfile")]
    public class DurianProfile
    {
        [Key]
        public int DurianId { get; set; }
        public string DurianName { get; set; }
        public string DurianDescription { get; set; }
        public string Characteristics { get; set; }
        public string TasteProfile { get; set; }
        public string DurianImage { get; set; }
        public int DurianVideoId { get; set; }

        public DurianVideo DurianVideo { get; set; }
        public ICollection<Seller> Sellers { get; set; } //To indicate many to many relationship
        public ICollection<User> Users { get; set; }//To indicate many to many relationship

        public List<FavoriteDurian> FavoriteDurians { get; set; } = new List<FavoriteDurian>();
    }
}

File: android_client/app/src/main/java/com/example/duriannet/models/DurianProfile.kt
package com.example.duriannet.models

data class DurianProfile(
    val durianId: Int,
    val durianName: String,
    val durianCode: String,
    val durianDescription: String,
    val characteristics: String,
    val tasteProfile: String,
    val durianImage: String,
    val durianVideoUrl: String,
    val durianVideoDescription: String
)
